cellprofiler/gui/help.py

```diff
@@ -2129,15 +2134,48 @@
     <li>Standard movie formats: AVI, Quicktime MOV, etc</li>
 </ul>CellProfiler uses the Bio-Formats library for reading various image formats. For more details on supported files,
 see this <a href="http://www.openmicroscopy.org/site/support/bio-formats4/supported-formats.html">webpage</a>. In
-general, we recommend saving stacks and movies in a format such as .TIF.
+general, we recommend saving stacks and movies in .TIF format.
+<p><i>Example:</i> You have several image stacks in the following format:</p>
+<ul>
+    <li>The stacks are saved in .TIF format.</li>
+    <li>Each stack is a single-channel grayscale image.</li>
+    <li>Your files have names like IMG01_CH01.TIF, IMG01_CH02.TIF, ... IMG01_CH09.TIF and IMG02_CH01.TIF,
+    IMG02_CH02.TIF, etc, where IMG01_CH01.TIF designates channel 1 from image 1, IMG01_CH02.TIF designates channel 2
+    from image 1, and IMG02_CH01.TIF designated channel 1 from image 2.</li>
+</ul>
+<p>In this case, the procedure to set up the input modules to handle these files is as follows:</p>
+<ul>
+    <li>In the <b>Images</b> module, drag-and-drop your folders of images into the File list panel. If necessary, set
+    your rules accordingly in order to filter out any files that are not files that are not images to be processed.<br>
+    In the above example, you would drag-and-drop the .TIF files into the File list panel.</li>
+    <li>In the <b>NamesAndTypes</b> module, select "Yes" for "Volumetric" to tell CellProfiler your images should be
+    processed as whole volumes. You can provide the voxel spacing in each dimension. CellProfiler will use this
+    information to correctly computer filter sizes and computing shape features, for example.<br>
+    Additionally assign each channel to a name of your choice. You will need to do this for
+    each channel. For this example, you could do the following:
+        <ul>
+            <li>Select "Assign images matching rules".</li>
+            <li>Make a new rule <code>[File][Does][Contain][CH01]</code></li>
+            <li>Provide a descriptive name for the channel, e.g., <i>DAPI</i>.</li>
+            <li>Click the "Add another image" button to define a second image with a set of rules.</li>
+            <li>Make a new rule <code>[File][Does][Contain][CH02]</code></li>
+            <li>Provide a descriptive name for the channel <i>GFP</i>.</li>
+            <li>Click the "Update" button below the divider to confirm that the proper images are listed and
+            matched across the channels. All file names ending in CH01.TIF should be matched together.</li>
+        </ul>
+    </li>
+    <li>In the <b>Groups</b> module, select "Yes" to automatically group your image sets by image name. All images
+    beginning with IMG01 will be processed in a single group, while images beginning with IMG02 will be processed in
+    another group, and so on. This way, CellProfiler can process all channels from each image.
+    </li>
+</ul>
```

Comment:
@bethac07 @karhohs @AnneCarpenter Can you please verify the above example makes sense? Lines 2137-2171.